manifests/kots-app.yaml

```diff
@@ -24,9 +24,9 @@ spec:
     - deployment/carto-workspace-subscriber
     - deployment/carto-workspace-www
     - '{{repl if ConfigOptionEquals "externalRedisEnabled" "0" }}deployment/carto-redis{{repl else}}{{repl end }}'
-  minKotsVersion: 1.107.4
+  minKotsVersion: 1.117.3
   additionalImages:
-    - gcr.io/carto-onprem-artifacts/tenant-requirements-checker:2024.4.17-rc.7
+    - gcr.io/carto-onprem-artifacts/tenant-requirements-checker:2025.6.10
```

Comment:
Do we have automatized in some way the auto-update of that image version on that file? if not it could cause us problems 🙈